communications/slack.md

```diff
@@ -0,0 +1,91 @@
+## On Slack
+
+Slack is a resourceful tool being used by a complex company. Each team will have specific dynamics that are from the scope of their tasks and it might require diverse communication tools.
+
+This is a gathering of some of the most used resources in slack to efficiently communicate based on takes from different people within the company. 
+
+Have in mind who you're trying to reach: If the person you want to correspond with is a highly contested resource, the expectation from them to engage with you synchronously will be frustrating. That's why we say `default to async communication`. 
+
+**Opt-in**: Join channels that you are interested in or obligated to.
```

Comment:
with the emphasis on thread usage, this can probably be generalized as

> Join ~~channels~~ conversations that you're interested in or obligated to.